include/nil/actor/zk/commitments/polynomial/fri.hpp

```diff
@@ -154,6 +154,8 @@ namespace nil {
                             evals_map, combined_U, combined_V,
                             transcript
                     );
+
+                    return true;
```

Comment:
As I understand we didn't reach this code.

Reply:
Removed, thanks for this one!